src/modules/company/controllers/deleteAdmin.js

```diff
@@ -49,50 +30,40 @@ export const deleteAdminFromCompany = async (databasePrisma, req, res) => {
   if (!userExists) {
     return res.status(400).send({ message: "User doesn't exist" });
   } else if (userExists.companyId !== id) {
-    return res
-      .status(400)
-      .send({ message: "User doesn't belong to this company" });
+    return res.status(400).send({ message: "User doesn't belong to this company" });
   }
 
   if (companyExists.admin.length === 1) {
-    return res
-      .status(400)
-      .send({ message: "A company requires at least 1 admin." });
+    return res.status(400).send({ message: "A company requires at least 1 admin." });
   }
```

Comment:
If the user id is wrong their access token will have thrown an error in the validation middleware, and verify access middleware will have thrown an error if their company id doesn't match.